### PR TITLE
add ansi with blocker layout and keymap for KBD67 rev2 PCB

### PR DESCRIPTION
**Pull request by elmo-space**
## Description

The rev2 pcb gets used in the kbd67 which has a blocker between the left arrow key and the right ctrl key. This layout is missing so far even though it's probably the most used one for this board.

Changes are tested on a board (thanks to someone on discord)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by elmo-space** [comment]
added another layout (on request by someone who helped me test this stuff) for blocker layout with split backspace.
The changes are now phyiscally tested on a board as well and seem to work just fine.

**Comment by zvecr** [comment]
https://github.com/qmk/qmk_firmware/blob/b936048b0bf2b5508104e243089e785debc431d0/keyboards/kbdfans/kbd67/rev2/rules.mk#L35

Should be updated to:
```makefile
LAYOUTS = 65_ansi 65_iso 65_ansi_blocker
```

**Comment by elmo-space** [comment]
apparently the 65_ansi map for this board does not follow standards as it breaks some of the community keymaps (like that of mechmerlin)

File: keyboards/kbdfans/kbd67/rev2/rules.mk [hunk 1]
```diff
@@ -32,4 +32,4 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
 
-LAYOUTS = 65_ansi 65_iso
+LAYOUTS = 65_ansi 65_iso 65_ansi_blocker 65_ansi_blocker_splitbs
```
Comment (zvecr) [inline keyboards/kbdfans/kbd67/rev2/rules.mk]: `65_ansi_blocker_splitbs` isnt an existing community layout
```suggestion
LAYOUTS = 65_ansi 65_iso 65_ansi_blocker
```

**Pull request merged**